src/geo/lng_lat.js

```diff
@@ -73,6 +73,40 @@ class LngLat {
         return `LngLat(${this.lng}, ${this.lat})`;
     }
 
+    /*
+    * Approximate radius of the earth.
+    * Uses the WGS-84 approximation. The radius at the equator is ~6378137 and at the poles is ~6356752. https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84
+    * 6371008.8 is one published "average radius" see https://en.wikipedia.org/wiki/Earth_radius#Mean_radius, or ftp://athena.fsv.cvut.cz/ZFG/grs80-Moritz.pdf p.4
+    *
+    * @returns {number} Average radius of the earth in meters.
+    */
+    static earthRadius(): number {
+        return 6371008.8;
+    }
```

Comment:
Nit: I wonder if we need to expose this publicly as a static method? Maybe just do `export const earthRadius = 6371008.8;` and import it (alongside the default export) in `MercatorCoordinate`?

Reply:
I like it. Done.